django/db/backends/base/creation.py

```diff
@@ -63,13 +70,16 @@ def create_test_db(self, verbosity=1, autoclobber=False, serialize=True, keepdb=
         # We report migrate messages at one level lower than that requested.
         # This ensures we don't get flooded with messages during testing
         # (unless you really ask to be flooded).
-        call_command(
-            'migrate',
-            verbosity=max(verbosity - 1, 0),
-            interactive=False,
-            database=self.connection.alias,
-            run_syncdb=True,
-        )
+        if self.connection.settings_dict.get("TEST", {}).get("MIGRATE", True):
+            call_command(
+                'migrate',
+                verbosity=max(verbosity - 1, 0),
+                interactive=False,
+                database=self.connection.alias,
+                run_syncdb=True,
```

Comment:
Does Django's test suite work with this new option? `run_syncdb=True` is the way we are hacking around not requiring migrations for Django's test suite, so if so, I think we can remove this option and the related code in `migrate`.